API/users/models.py

```diff
@@ -1,2 +1,38 @@
+from django.db import models
+from django.contrib.auth.models import User
 
 # Create your models here.
+
+class Account(models.Model):
+    # account_set added implicitly
```

Comment:
are you talking about product group set?

Reply:
yes, sorry